Blazorade.Bootstrap.Components.Showroom/ProgressBars.razor

```diff
@@ -0,0 +1,134 @@
+﻿@code {
+    Spacing headingMarginTop = Spacing.Five;
+}
+
+<Heading Level="HeadingLevel.H2">Progress/ProgressBar Component</Heading>
+<Paragraph>
+    The <Anchor Url="https://getbootstrap.com/docs/4.4/components/progress/" OpenInNewTab="true">Progress component</Anchor>
+    is used to display progress bars, including stacked bars, animated backgrounds, and text labels.
+</Paragraph>
+
+<DocsSection ComponentName="ProgressBar" />
+
+<Heading Id="basic-progress-bar" IsAnchor="true" Level="HeadingLevel.H3" MarginTop="@headingMarginTop">Basic Progress Bars</Heading>
+<Paragraph>
+    A basic progress bar example.
+</Paragraph>
+
+<Progress>
+    <ProgressBar MaxValue="100" MinValue="0" Value="0"></ProgressBar>
+</Progress>
+&nbsp;
+<Progress>
+    <ProgressBar MaxValue="100" MinValue="0" Value="25"></ProgressBar>
+</Progress>
+&nbsp;
+<Progress>
+    <ProgressBar MaxValue="100" MinValue="0" Value="50"></ProgressBar>
+</Progress>
+&nbsp;
+<Progress>
+    <ProgressBar MaxValue="100" MinValue="0" Value="75"></ProgressBar>
+</Progress>
+
+
+<Heading Id="progress-bar-labels" IsAnchor="true" Level="HeadingLevel.H3" MarginTop="@headingMarginTop">Labels</Heading>
+<Paragraph>
+    Add text inside of the ProgressBar component to add labels to a progress bar.
+</Paragraph>
+
+<Progress>
+    <ProgressBar MaxValue="100" MinValue="0" Value="25">25%</ProgressBar>
```

Comment:
Could we add a `ShowPercentage` parameter or something like that to define that the component would calculate the percentage automatically?

It could work so that if the `ChildContent` is not specified for the component, but the `ShowPercentage` is set, then the component would render the percentage automatically using integer values (no decimals so that we don't have to deal with localization issues). A developer could always override that by simply specifying child content for the component.

Reply:
Sure, can do.